### PR TITLE
add user model

### PR DESCRIPTION
**Pull request by chirag-shinde**
fixes #51

### CODE REVIEW

File: server/src/models/user.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  picture: {
+    type: String,
+  },
+});
+
+const User = mongoose.model('User', userSchema);
+
+function validateUser(user) {
+  const joiSchema = {
+    given_name: Joi.string().min(2).max(50).required(),
```
Comment (nimish-gupta) [inline server/src/models/user.js]: Which typecase are you guys following?

**Review by nimish-gupta — APPROVED** [review]

File: server/src/models/user.js (hunk 1 above)
Comment (chirag-shinde) [reply server/src/models/user.js]: camelcasing for variable names,
PascalCase for component/class naming

Comment (chirag-shinde) [reply server/src/models/user.js]: will update the property names with an underscore to camelcase.

File: server/src/models/user.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    family_name: Joi.string().min(2).max(50).required(),
```
Comment (Prabhakar-Poudel) [inline server/src/models/user.js]: family name is not required in the above schema. Will that be an issue?

**Review by Prabhakar-Poudel — APPROVED** [review]
LGTM

File: server/src/models/user.js (hunk 2 above)
Comment (chirag-shinde) [reply server/src/models/user.js]: I have removed the required constraints.
Only the email is required now.

**Pull request merged**